### DISCUSSION TITLE
Getting “Access Violation at address 98F869FC, accessing memory address 00000000”

### DISCUSSION THREAD

**Discussion by blasco403** [GerryDazoo/SlingerPlayer#8] [Q&A]
I am getting the above error with SlingerPlayer on a 3rd gen FireTV Lite. I think it may be related to a recent "System Components" update the FireOS did recently.

Screen goes white for a minute or so and then produces that error while streaming. Doesn't seem to consistent with streaming duration.

Anyone else experiencing this?

Just the normal session disconnect message on Slinger Server. 

Thanks

**Comment by GerryDazoo** [comment]
I just got that same exception after a few seconds of streaming but in my case the stream continued to run after a few hesitations. I don't have time to investigate until tomorrow but at this point in the streaming the code I wrote isn't being executed. It's all down in the guts of the ffmpeg libraries that are doing the heavy lifting. Hopefully I'll be able to sort it out.

**Reply by blasco403** [reply]
Thanks Gerry. Just went back to kodi for now. No rush!!
